### ISSUE TITLE
SIP protection and dumping ALL the services.

### ISSUE THREAD

**Issue by SKGrimes** [drduh/OS-X-Security-and-Privacy-Guide#58]
I noticed the SIP section included in the guide but it doesn't have much on disabling it and since SIP is enabled by default. Obviously, it is super trivial to just type:

	# csrutil disable
	
At the bash prompt, however if you do not do it you can't follow the guide. None of the changes will hold, if they are even possible. 

* All of the `luanchctl ` commands fail and can not find any services running
* All of the roots can not be removed without disabling
* No applications that are "essential" can be removed without disabling it.
* Root directory access is limited and read only for many directories including /etc

After removing it the following programs can be deleted:
1. messages.app 
   * may need to rm -rf chatdb by force as it runs recursively and prevented trash to delete it.
2. game center.app
3. chess.app
4. photo booth.app

If you dump too many programs or services the OS WILL destabilize. I unloaded many services as well. I unloaded all of the ones in the guide without *too many* issues. 

**Stability** Other services and daemons (and possibly some of the below) can cripple and destabilize the OS. It seems to rely a lot more heavily on the core services and CPU spikes seem to occur due to persistant connection attempts or program requirements. 

**Firmware**You can not dump ram with this PW enabled, which is a bitch. This has always been the case, but it isn't mentioned. If there is a way to do it, would love to know.

**Dump unused Languages** You CAN NOT dump unused languages with monolingual unless you are running rootless.

tl;dr please include `csrutil disable` in the guide as running does not seem possible without doing that first.
